server/service/endpoint_appconfig.go

```diff
@@ -92,6 +95,7 @@ func makeGetAppConfigEndpoint(svc fleet.Service) endpoint.Endpoint {
 			HostSettings:       hostSettings,
 			License:            license,
 			AgentOptions:       agentOptions,
+			Logging:            fleet.LoggingFromConfig(svc.FleetConfig(ctx)),
```

Comment:
I think it would be simpler if this would be `svc.LoggingConfig(ctx)`. The only reason to expose this config at this point is to get the logging bits, and we are doing it with a method anyway. What do you think?

Reply:
@chiiph when @zwass and I spoke I think it made sense to expose the entire FleetConfig in case we need to pull something from it in the near future. At the moment some of the logging bits are only relevant.

Reply:
I know we spoke about including the entire config, however thinking about it now I have a bit of concern that we could at some point in the future end up unintentionally exposing secrets with this `svc.FleetConfig` method. Maybe requiring that a new method be created to expose additional config could add a bit of protection against that?

I think moving the `LoggingFromConfig` configuration copying logic into the proposed `svc.LoggingConfig` function could strike a good balance.

Reply:
great suggestions, makes sense to me